fern/pages/docs/getting-started/quickstart.mdx

```diff
@@ -1,6 +1,7 @@
 ---
 title: Quickstart
-description: Start building beautiful documentation in under 5 minutes.
+description: Start building beautiful documentation in under 5 minutes
```

Comment:
pretty sure just setting subtitle will accomplish this without the duplication

Reply:
gotcha